internal/visualization/console/feature_map.go

```diff
@@ -0,0 +1,28 @@
+package console
+
+import (
+	"golang.org/x/mod/semver"
+	"strings"
+)
+
+const (
+	featureDevConsole = "dev-console"
+)
+
+var (
+
+	// featuresIfUnmatched represents the default features to enable
+	featuresIfUnmatched = []string{featureDevConsole}
```

Comment:
Here an further on there seems to be a mix of terms - matching vs comparison.

Reply:
I don't understand what you are asking or suggesting